servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java

```diff
@@ -79,6 +80,28 @@ public void getBlockSizeBytesTest() {
   }
 
   @Test
+  public void getBlockIdByIndexTest() throws Exception {
+	  InodeFile inodeFile = createInodeFile(1);
+	  // create 3 block ids
+	  inodeFile.getNewBlockId();
```

Comment:
Done, but broken into two for loops so that an implementation of "always return the last block id" doesn't pass.